src/main/cli/cli.c

```diff
@@ -165,6 +165,7 @@ extern uint8_t __config_end;
 
 #include "cli.h"
 
+#include "platform.h"
```

Comment:
Why are you including `platform.h` for a second time.

Reply:
Oops.